Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -160,20 +155,9 @@ export class azureclitask {
             return;
         }
 
-        var configDirName: string = "c" + new Date().getTime(); // 'c' denotes config
-        var basePath: string;
-        if (tl.osType().match(/^Win/)) {
-            basePath = process.env.USERPROFILE;
-        }
-        else {
-            basePath = process.env.HOME;
-        }
-
-        if (!!basePath) {
-            this.azCliConfigPath = path.join(basePath, ".azclitask", configDirName);
-            console.log(tl.loc('SettingAzureConfigDir', this.azCliConfigPath));
-            process.env['AZURE_CONFIG_DIR'] = this.azCliConfigPath;
-        }
+        var azCliConfigPath = path.join(tl.getVariable('Agent.TempDirectory'), ".azclitask");
```

Comment:
is it better to check whether this variable exists? This was added in 2.115.0 version of agent ans hence should be safe. But we can still check.

Reply:
done